examples/v1beta1/pipelineruns/pipelinerun.yaml

```diff
@@ -26,82 +14,165 @@ roleRef:
   name: cluster-admin
   apiGroup: rbac.authorization.k8s.io
 ---
-apiVersion: tekton.dev/v1alpha1
-kind: PipelineResource
-metadata:
-  name: skaffold-image-leeroy-web-pipelinerun
-spec:
-  type: image
-  params:
-  - name: url
-    value: gcr.io/christiewilson-catfactory/leeroy-web
----
-apiVersion: tekton.dev/v1alpha1
-kind: PipelineResource
-metadata:
-  name: skaffold-git-pipelinerun
-spec:
-  type: git
-  params:
-  - name: revision
-    value: v0.32.0
-  - name: url
-    value: https://github.com/GoogleContainerTools/skaffold
----
 apiVersion: tekton.dev/v1beta1
 kind: Task
 metadata:
   name: unit-tests
 spec:
-  resources:
-    inputs:
-    - name: workspace
-      type: git
-      targetPath: go/src/github.com/GoogleContainerTools/skaffold
+  workspaces:
+  - name: source
+    mountPath: /workspace/source/go/src/github.com/GoogleContainerTools/skaffold
   steps:
   - name: run-tests
     image: golang
     env:
     - name: GOPATH
       value: /workspace/go
-    workingDir: $(resources.inputs.workspace.path)
+    workingDir: $(workspaces.source.path)
     command:
     - echo
     args:
     - "pass"
```

Comment:
Opportunity here to use script mode, and comment that normally this would `go test ./...` but we don't want this to fail if skaffold pushes a bad change.

Reply:
by the way, we could use `github.com/tektoncd/pipeline` here instead, so that we don't depend on anyting but us

Reply:
Using skaffold might not be as unreliable as we think, since we've tied ourselves to the v0.32.0 release

So maybe longterm the answer is to update it to use our own tests, also tied to a release. :D

And now that we're tied to a revision maybe we can actually run the tests again 🤔